packages/google-cloud-bundle/composer.json

```diff
@@ -18,10 +18,10 @@
     "require": {
         "php": "^7.1",
         "google/cloud-storage": "^1.10",
-        "shopsys/form-types-bundle": "@dev",
-        "shopsys/framework": "@dev",
-        "shopsys/migrations": "@dev",
-        "shopsys/plugin-interface": "@dev",
+        "shopsys/form-types-bundle": "7.0.0-beta6",
```

Comment:
will this be replaced into @dev ? https://github.com/shopsys/shopsys/pull/730#discussion_r247502348

Reply:
Yes, there is worker for changing back.

Reply:
I would not expect it to happen automatically (unless the releaser was changed in the linked PR). Was this change made manually or automatically? :slightly_smiling_face:

Reply:
this package is new and was not in beta5 but @dev -> 7.0.0-beta6 i think that this will be changed back into dev-master rather than @dev, but we will see

Reply:
Oh... I'm a bit confused right now. So the proper state after the release should be `@dev`? 
And if releaser changes it to `dev-master`, I should manually change it to @dev?

Reply:
yes, I would expect that it will be changed to `dev-master`, so it would be nice to change it manually to `@dev` and create an issue for fixing releaser...Moreover, there is a question (that could be a part of the issue) - do we want to use `@dev` everywhere? Besides, there was an issue that we could start using branch aliasing, however, I am not able to find now :confused: